lma-websocket-stack/deployment/lca-websocket.yaml

```diff
@@ -1520,6 +1527,7 @@ Resources:
                         - !GetAtt WebSocketCloudFrontDistribution.DomainName
                         - '/api/v1/ws'
       recordingDisclaimer: !Ref RecordingDisclaimer
+      assistantWakePhraseRegEx: !Ref AssistantWakePhraseRegEx
```

Comment:
worth adding a comment to the template to say why we added this property - otherwise some diligent person might reasonably conclude that since it's not actually used in the Lambda it can be removed.

Reply:
+1